vortex-array/src/canonical.rs

```diff
@@ -89,6 +89,18 @@ impl Canonical {
             }
         }
     }
+
+    /// Unwrap Canonical Array back into a regular Array
+    pub fn into_array(self) -> Array {
```

Comment:
why this vs just implementing the `into_array` trait for Canonical?

Reply:
should do just that

Reply:
This already exists